src/applications/simple-forms/form-upload/helpers/index.js

```diff
@@ -78,3 +78,14 @@ export const onCloseAlert = e => {
 export const getMockData = (mockData, isLocalhost) => {
   return !!mockData && isLocalhost() && !window.Cypress ? mockData : undefined;
 };
+
+export const formattedPhoneNumber = phoneNumber => {
+  const phoneNumberWithoutDashes = phoneNumber.replaceAll('-', '');
+  return `(${phoneNumberWithoutDashes.substr(
+    0,
+    3,
+  )}) ${phoneNumberWithoutDashes.substr(
+    3,
+    3,
+  )}-${phoneNumberWithoutDashes.substr(6, 4)}`;
+};
```

Comment:
This can be simplified if you're comfortable with regex:

```suggestion
export const formattedPhoneNumber = phoneNumber => {
  const digits = phoneNumber.replaceAll('-', '');
  return digits.replace(/(\d{3})(\d{3})(\d{4})/, '($1) $2-$3');
};
```